src/app/providers/profile/profile.ts

```diff
@@ -155,6 +171,69 @@ export class ProfileProvider {
     }
   }
 
+  public setWalletGroupsHome(walletObj) {
+    let result = _.cloneDeep(this.resultPrimaryAccount);
+    let walletsGroupsHome = [];
+    walletsGroupsHome = JSON.parse(localStorage.getItem("listHome")) ? JSON.parse(localStorage.getItem("listHome")) : [];
+    if (walletObj && walletObj.walletId) {
+      if (walletsGroupsHome.length < 5) {
```

Comment:
max wallet added in home is 5 why checking length < 5

Reply:
when array:
- 4 item =>  array.length=4 < 5 => added
- 5 item =>  array.length=5 <= 5 => added
===> keep condition < 5